web/blackjack/src/app/game/game/game.component.ts

```diff
@@ -38,18 +47,29 @@ export class GameComponent implements OnInit {
     });
   }
 
+  onReady(): void {
+    this.blackjackService.readyPlayer(this.gameId, this.playerId);
+  }
+
   onStartGame(): void {
     this.blackjackService.startGame(this.gameId);
   }
 
   private fetchGame(): void {
-    this.blackjackService.getGame(this.gameId, this.playerId).subscribe((response) => {
-      this.you = this.createPlayer(response.you, this.playerId);
-      this.otherPlayers = response.players.map((otherPlayerMetadata) => this.createPlayer(otherPlayerMetadata, undefined));
-      this.hasStarted = response.has_started;
-      if (response.results) {
-        this.results = new ResultsModel(response.results.result, this.createPlayer(response.results.winning_player));
-      }
+    if (this.isReady) {
+      this.blackjackService.getGame(this.gameId, this.playerId).subscribe((response) => {
+        this.you = this.createPlayer(response.you, this.playerId);
+        this.otherPlayers = response.players.map((otherPlayerMetadata) => this.createPlayer(otherPlayerMetadata, undefined));
+        this.hasStarted = response.has_started;
+        if (response.results) {
```

Comment:
Somehow these results are not showing up, I'm not sure why, though. I assume that because the `fetchGame()` is run every 500ms, results should show once `isReady == true` and results are available. Can you try to help me find why this isn't working?

Reply:
From what I can tell, when I run this locally, it works for me. We can review this together this morning and see what the issue is.